mautic/mautic
# fix: [DPMMA-2661] Correct form mapper Mautic 5

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🟢 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description
Bug improves pool mapping via API Endpoint /api/forms/{form_id}/edit
The error causes the form fields to not be mapped correctly to the contact or company.
An incorrectly mapped field caused contacts to remain anonymous.







---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Send via API from /api/forms/{form_id}/edit with mapped field (leadField - deprecated or pair mappeField, mappedObject)
